internal/safeexpvar/cacheexpvar_test.go

```diff
@@ -0,0 +1,33 @@
+package safeexpvar
+
+import (
+	"expvar"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSetExpvarInt(t *testing.T) {
+	initialGoroutines := runtime.NumGoroutine()
```

Comment:
just add a TestMain  like this:

https://github.com/jaegertracing/jaeger/blob/590036037122082b46d830e4c6caa299f7fdbe92/pkg/version/package_test.go#L12_L14